package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@vernier/vst-video-utils-build",
+  "version": "1.3.0",
+  "description": "Container of prebuilt binaries used for FFmpeg.",
+  "main": "VideoUtils.js",
+  "module": "VideoUtils.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "release": "release-it"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/VernierST/vst-video-utils-build.git"
+  },
+  "author": "Vernier Software and Technology",
+  "license": "LGPL-3.0",
+  "bugs": {
+    "url": "https://github.com/VernierST/vst-video-utils-build/issues"
+  },
+  "homepage": "https://github.com/VernierST/vst-video-utils-build#readme",
+  "devDependencies": {
+    "release-it": "^13.0.2",
+    "@vernier/vst-video-utils-build": "latest"
```

Comment:
Just curious why you have this as a devDependency?

Reply:
I figured since we were combining into a single package.json instead of 2, we needed it in there.  Maybe I am misunderstanding, I can pop it out.

Reply:
Yeah, I don't think we need it here 👍